gilbarbara/react-joyride
# Add `allowClicksThruHole` prop & step prop

Fixes #189 

## Description

This fixes the breaking change which was introduced by https://github.com/gilbarbara/react-joyride/pull/157.  

It restores the behavior of version 1.7.0 of simply firing a callback when the hole is clicked, unless the prop `allowClicksThruHole` is set to `true` on the Joyride or on the step.

## To Test

Remove `allowClicksThruHole` from the step in the demo, and set `disableOverlay` to `false` and `debug` to `true`.  Verify that it is not possible to click the "Advance" button, and instead the callback `hole:click` is fired.
